docs/tutorials/ionq/getting_started.ipynb

```diff
@@ -0,0 +1,328 @@
+{
```

Comment:
<p>are these opening two code snippets supposed to be here? seems like an accidental paste.</p><p> </p><p>Also, also an API <em>key. </em>Accidentally a word there.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/3758/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>ugh not sure how that happened.  fixed and fixed missing key</p>